code/modules/surgery/organs/dwarf_organ.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/organ/alcoholvessel //essentially the opposite of the xeno's plasmavessel, but with alcohol
```

Comment:
@RemieRichards You are supposed to be able to rip a dwarf apart, extract their booze vessel, and insert it into yourself to gain their booze dependency.